### DISCUSSION TITLE
GUI interface?

### DISCUSSION THREAD

**Discussion by PalZer0** [UMSKT/UMSKT#133] [Ideas]
Are there plans for a GUI interface to make UMSKT easier to work with (especially for the function of generating confirmation IDs)?

**Comment by CONIGUERO** [comment]
> Are there plans for a GUI interface to make UMSKT easier to work with (especially for the function of generating confirmation IDs)?

In the long run, yes.

For now, you can use a third-party web GUI, https://mskt.surge.sh, based on https://github.com/anpage/umskt-rs and made by the same person.

**Reply by TheSystemGuy1337** [reply]
Small update, MSKT appears to be down.

**Comment by thepwrtank18** [comment]
https://mskt.surge.sh

This is a web-based implementation, however, it's not official and might be behind on updates.

Edit: damn I was late

**Comment by PalZer0** [comment]
Good to hear that it's planned. I'm not in any rush for one so feel free to take your time.

**Comment by Neo-Desktop** [comment]
I think it's safe to say we will have a minimalist GUI, but @Endermanch will have a keygen aesthetic gui for his version
